src/components/Board.js

```diff
@@ -3,29 +3,39 @@ import './Board.css';
 import Square from './Square';
 import PropTypes from 'prop-types';
 
-
+// This turns the 2D array into a 1D array
 const generateSquareComponents = (squares, onClickCallback) => {
-  // Complete this for Wave 1
-  // squares is a 2D Array, but 
-  //  you need to return a 1D array
-  //  of square components
-
-}
+  const singleArraySquares = [].concat(...squares);
+  return singleArraySquares.map((square) => {
+    return (
+      // map returns a square component for each object
+      // each one gets a value, id, onClickCallBack, and a unique key
+      <Square
+        value={square.value}
+        id={square.id}
+        onClickCallback={onClickCallback}
+        key={square.id} // just a unique identifier that React needs
+      />
+    );
+  });
+  // returns an array of square components
+};
 
 const Board = ({ squares, onClickCallback }) => {
   const squareList = generateSquareComponents(squares, onClickCallback);
   console.log(squareList);
```

Comment:
I think we left this for debugging purposes for you, but let's always get rid of any print statements before final submission
```suggestion
```